src/helper/client.cpp

```diff
@@ -152,6 +157,7 @@ bool client::handle_command(const network::client_init::request &command)
     bool has_errors = false;
 
     client_enabled_conf = command.enabled_configuration;
+    compute_client_status();
```

Comment:
`client_init` lies outside of the request lifecycle so I think this might not be needed here.